README.md

```diff
@@ -376,7 +376,7 @@ Commands:
 
 Available models:
 
-* disease_mesh
```

Comment:
disease mesh is also available no?

Reply:
well I am replacing it with the entire mesh, we can make both available but we have created them through `dvc repro` so there is a minor chance there might be incompatible